test/verify/check-journal

```diff
@@ -388,7 +388,10 @@ s.send("PRIORITY=3\\nFOO=bar\\n")'
         m.execute("setenforce 0; ulimit -c unlimited; echo 'sleep 42m &' > slp; chmod u+x slp")
         m.execute("./slp; pkill -SEGV sleep")
```

Comment:
This should be `pkill -x`, no?  The unpredictable function name comes from accidentially killing not just `sleep`, but also `pmsleep`, I think.